transmogrifier/sources/json/aardvark.py

```diff
@@ -35,7 +54,7 @@ def get_source_record_id(cls, source_record: dict) -> str:
         Args:
             source_record: A JSON object representing a source record.
         """
-        return source_record["id"]
+        return source_record["id"].replace("mit:", "").replace("ogm:", "")
```

Comment:
Much cleaner, thanks!

Reply:
Always a fan of regex, and could still have a place here, but just noting this would also strip `mit:` or `ogm:` from the middle of the identifier if it somehow existed there as well ([related comment](https://github.com/MITLibraries/transmogrifier/pull/110#discussion_r1442117882)).